packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-blocks-manage-button.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { isReusableBlock } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
+import { BlockSettingsMenuControls } from '@wordpress/block-editor';
+import { addQueryArgs } from '@wordpress/url';
+
+function ReusableBlocksManageButton( { clientId } ) {
+	const { isVisible } = useSelect(
+		( select ) => {
+			const { getBlock } = select( 'core/block-editor' );
+			const { canUser } = select( 'core' );
+			const blockObj = getBlock( clientId );
```

Comment:
Would be great to avoid an abbreviation here (the coding standards says to use full names where possible - https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions).

I think I'd go with naming this `reusableBlock` and change the later variable to `reusableBlockPost` (if it's needed).